cumulusci/tasks/github/publish.py

```diff
@@ -20,8 +24,12 @@ class PublishSubtree(BaseGithubTask):
             "required": True,
         },
         "version": {
-            "description": "The version number to release. "
-            "Also supports latest and latest_beta to look up the latest releases. "
+            "description": "(Deprecated) Only the values of 'latest' and 'latest_beta' are acceptable. "
```

Comment:
±: I would argue in favor of a specific deprecation version, say `>= 3.44.0`?

Reply:
Current release is 3.41.0. So should we say `(Deprecated >= 3.42.0)`?